src/plugins/translation-inject.js

```diff
@@ -17,14 +17,17 @@ module.exports = (opts = {}) => {
       const magicString = new MagicString(code);
       this.parse(code, {
         onComment: (block, text, start, end) => {
-          const match = text.match(/@ngTranslationsInject(.*)/);
+          const match = text.match(/@ngTranslationsInject:?([a-zA-Z]+)?(.*)/);
           if (match) {
             const translations = _.chain(match)
-              .get(1)
+              .get(2)
               .split(/\s+/)
               .value();
+
+            const format = match[1];
```

Comment:
_.get(match, "1") for consistency

Reply:
Yes, i saw the idea, but i think your suggestion is false ^^

Should be `_.get(match, '[1]')`, right ?

Reply:
yep :)